### DISCUSSION TITLE
Subset ArchR multiome project

### DISCUSSION THREAD

**Discussion by millerkaplanlab** [GreenleafLab/ArchR#1797] [Questions / Documentation]
Is there any information with regards to subsetting an ArchR mutiome project? I would like to extract specific cell types (based on scRNA-seq data); however, I am not sure how to proceed after extracting the barcodes for these clusters.

**Comment by rcorces** [comment]
There should be no difference in subsetting a multiome project compared to an ATAC-only project.

**Reply by millerkaplanlab** [reply]
Thank you for your reply. I'm just a bit confused as to where to proceed with the pipeline after subsetting the clusters/barcodes I want to keep.

**Reply by rcorces** [reply]
There really isnt any difference between a multiome project and an ATAC-only project. The primary difference would be that instead of using `GeneIntegrationMatrix` in certain analyses, you would use `GeneExpressionMatrix`. I've put together a new multiome tutorial which will soon become part of the main manual. That is currently visible here - https://github.com/rcorces/ArchR_Website_Testing/blob/master/bookdown/21_MultiomicDataAnalysis.Rmd

but I dont think it will provide any explicit information for you.

**Reply by millerkaplanlab** [reply]
Thank you for this information. I am currently trying to use the subsetArchR function; however, I am unsure about how to select for specific clusters. Any help would be appreciated. Thank you.
